lib/parameters.py

```diff
@@ -147,6 +147,23 @@ def gen_avc_vbr_lp_parameters(spec, profiles):
   params = gen_avc_vbr_lp_variants(spec, profiles)
   return keys, params
 
+def gen_avc_vbr_la_variants(spec, profiles):
+  for case, params in spec.iteritems():
+    for variant in copy.deepcopy(params.get("vbr_la", [])):
+      uprofile = variant.get("profile", None)
+      cprofiles = [uprofile] if uprofile else profiles
+      for profile in cprofiles:
+        yield [
+          case, variant["bframes"], variant["bitrate"], variant.get("fps", 30),
+          variant.get("quality", 4), variant.get("refs", 1),
+          profile, variant["lookahead"], variant["lookaheaddepth"]
```

Comment:
`lookahead` parameter is unnecessary since it is implied with the `vbr_la` list.  Please remove `lookahead`.

Also, can we use `la_depth` or `ladepth` instead of `lookaheaddepth`?  This should also be an optional setting with reasonable default value like we do for fps, refs and quality.